java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -672,6 +674,31 @@ public T unlink(String[] keys) {
         return getThis();
     }
 
+    /**
+     * Rewrite the configuration file with the current configuration.
+     *
+     * @see <a href="https://redis.io/commands/config-rewrite/">redis.io</a> for details.
+     * @return <code>OK</code> when the configuration was rewritten properly, otherwise an error is
```

Comment:
no error will be raised. please check the documentation - sometimes there's a difference between the client docs and transaction docs.